### DISCUSSION TITLE
Svelte example, API not working locally

### DISCUSSION THREAD

**Discussion by msurguy** [vercel/vercel#4133] [General] [answered]
When I'm trying out Svelte example using `npm run dev`, the API part of the demo is not working as intended, it shows the following instead of the date:

<img width="822" alt="Screen Shot 2020-04-23 at 10 25 33 AM" src="https://user-images.githubusercontent.com/585833/80130147-00c35a00-854d-11ea-8d30-df8397db5c5a.png">

What command do I need to run locally to get the Node.js portion to work right?

**Comment by mcsdevv** [comment, marked as answer]
Hey @msurguy, you'll need to use `now dev` to run this locally and have the Serverless Functions execute correctly.

**Reply by msurguy** [reply]
Thanks, @msweeneydev ! It took 3 tries of `now dev` to get the output I was looking for. On the first try it would link up the deployment and deploy, on the second try I got what's in the screenshot, and on the third try I got the correct output. 

<img width="617" alt="Screen Shot 2020-04-23 at 10 42 57 AM" src="https://user-images.githubusercontent.com/585833/80131653-67497780-854f-11ea-9676-d32eb5e380fa.png">

Thank you so much!

**Reply by mcsdevv** [reply]
Awesome, glad you got there, sorry it wasn't so smooth! I think in the near future we'll see a lot more frameworks (not just Next.js) implementing Serverless Functions that can be run with their own development command.

Until then, we're working hard on `now dev` to improve it greatly and provide an option for all.

**Reply by msurguy** [reply]
Thanks!
